sdk/resourcemanager/appservice/staticsite/README.md

```diff
@@ -1,13 +1,3 @@
----
-page_type: sample
```

Comment:
why do we remove this?

Reply:
Seems it is used for doc index. I restored it.